tests/testsuite/help.rs

```diff
@@ -147,7 +140,11 @@ fn help_alias() {
     // The `empty-alias` returns an error.
     cargo_process("help empty-alias")
         .env("PATH", Path::new(""))
-        .with_stderr_contains("[..]The subcommand 'empty-alias' wasn't recognized[..]")
+        .with_stderr_data("The subcommand 'empty-alias' wasn't recognized
+
+FIXME: #14076 This assertion isn't working, as this line should have caused a test failure but didn't.
+        ",
+        )
         .run_expect_error();
```

Comment:
```suggestion
        .with_status(101)
        .with_stderr_data(str![[r#"
[ERROR] no such command: `empty-alias`

	Did you mean `empty-alias`?

	View all installed commands with `cargo --list`
	Find a package to install `empty-alias` with `cargo search cargo-empty-alias`

"#]])
        .run();
```

Something like this should work. See <https://github.com/rust-lang/cargo/issues/14076#issuecomment-2169593172>.

(just realized that the error message regressed…)

Reply:
Would you mind posting another PR fixing #14076 and then update this afterward?

Reply:
Thanks! Just get `SNAPSHOTS=overwrite` working and pushed 8db30bc.

> Would you mind posting another PR fixing #14076 and then update this afterward?

Sure! Will address #14076 first with #14078, and then resolve the conflict here.